client/models/channel.coffee

```diff
@@ -77,7 +77,14 @@ module.exports = class Channel extends Backbone.Model
         'auctions'
       ], association
     else if type is 'partner'
-       false
+      _.contains [
```

Comment:
super small, but it looks like the elements in the array passed to `_.contains` are the same elements. maybe defining an associations array after line 58 could remove some duplication. nothing to hold up the PR tho. something to consider on the next one...